radixdlt/src/integration/java/com/radixdlt/consensus/deterministic/ControlledNode.java

```diff
@@ -67,34 +68,45 @@
 	private final ValidatorSet initialValidatorSet;
 	private final ControlledSender controlledSender;
 
+	public enum SyncAndTimeout {
+		NONE,
```

Comment:
is `NONE` better modelled using `Optional< SyncAndTimeout >` ? That's how I would do it in Swift, but maybe it would have syntactical overhead here in Java land, in Swift it does not have any overhead, but is the encourage way of doing it.

Reply:
Not part of PR.

Reply:
Optional, doesn't cover the three states required.